templates/tour.html

```diff
@@ -4,300 +4,301 @@
-                        <li><a href="https://docs.ubuntu.com/maas/2.2/en/intro-concepts#spaces" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });" class="external" aria-label="External link to intro concepts about spaces on MAAS documentation">Spaces</a></li>
-                        <li><a href="https://docs.ubuntu.com/maas/2.2/en/installconfig-network-ntp" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });" class="external" aria-label="External link to install and configure network ntp on MAAS documentation"><abbr title="Network Time Protocol">NTP</abbr></a></li>
-                    </ul>
-                </div>
-                <div class="two-col">
-                    <ul>
-                        <li><a href="https://docs.ubuntu.com/maas/2.2/en/installconfig-network-ssl" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });" class="external" aria-label="External link to install and configure secure sockets layer on MAAS documentation"><abbr title="Secure sockets layer">SSL</abbr></a></li>
-                        <li><a href="https://docs.ubuntu.com/maas/2.2/en/installconfig-network-stp" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });" class="external" aria-label="External link to install and configure spanning tree protocol on MAAS documentation"><abbr title="Spanning tree protocol">STP</abbr></a></li>
-                        <li><a href="https://docs.ubuntu.com/maas/2.2/en/installconfig-network-proxy" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Feature', 'eventAction' : 'Click', 'eventLabel' : 'Network management' });" class="external" aria-label="External link to install and configure network proxy on MAAS documentation">Proxy</a></li>
-                    </ul>
-                </div>
+            <div class="col-6">
+                <p>With full <abbr title="Domain name system">DNS</abbr> management you can:</p>
+                <ul>
+                    <li>Create multiple <abbr title="Domain name system">DNS</abbr> domains</li>
+                    <li>Add multiple records per domain</li>
+                    <li>Select the domain for machines and devices</li>
+                    <li>Assign additional names to <abbr title="Internet Protocol address">IP addresses</abbr> and more&hellip;</li>
+                </ul>
             </div>
+    </div>
+    <div class="row">
+            <h3>And much more</h3>
+            <p>MAAS can easily manage the most complex network environments, including all your:</p>
+                <ul class="col-2">
```

Comment:
Over indented. This one will require peters modifier to the split list pattern on Ubuntu.com